scripts/UpdateTextMate.ps1

```diff
@@ -0,0 +1,40 @@
+# This script updates our *.tmLanguage.json files from the latest copies in the VS Code repo.
```

Comment:
The scripts folder contains scripts that ship in the extension VSIX and are used by the extension at runtime.  I don't think this is needed at runtime right?  This should move somewhere else.

If its not too hard, the best place for this is a gulp task (if this ever needs to run in CI I think this needs to be gulp, not sure if all the linux machines have powershell).

Reply:
Ah, didn't realize the scripts folder is shipped. Yeah, UpdateTextMate.ps1 isn't needed at runtime. I moved it to the Razor tests folder since it's only used for tests at the moment. Currently it doesn't run in Razor's CI so I think it's fine to leave it as a Powershell script for now and convert it to a gulp task later if we decide we do want it in CI